### ISSUE TITLE
Launch Nukes 1 by 1

### ISSUE THREAD

**Issue by sashokbg** [beyond-all-reason/Beyond-All-Reason#1772]
Feature Request:

When having selected multiple nuke silos
I would like to be able to issue an attack command multiple times
So that each nuke lands on a different spot
And only one nuke is launched each time.

Example:

I have built 5 nuke silos
I select them all
I scrow to the enemy base
I click attack and i click 3 times my mouse on 3 different locations
Then only 3 nukes are launched

Instead of all 5 like it currently does on the first click

**Comment by Okema1** [comment]
For the most part you can already do this. If you have multiple nuke silos selected when attacking you can right click and drag and each nuke will get launched to different spots along were you have dragged.

**Comment by sprunk** [comment]
dragging paces them all along your line though so you can't conveniently do a weird shape like this with 5 silos (both because there's less nukes you want to launch and because the shape is not a neat line)
![image](https://github.com/beyond-all-reason/Beyond-All-Reason/assets/2573076/907404f5-56bf-4af8-86af-944dd4e12a38)

**Comment by sashokbg** [comment]
I agree with @sprunk I just completed scenario "Outsmart the barbarians" and the AI normally spreads everywhere on this map so I want my nukes to cover as much as possible
